docs/content.zh/docs/dev/table/functions/udfs.md

```diff
@@ -1282,7 +1282,7 @@ public static class WeightedAvg extends AggregateFunction<Long, WeightedAvgAccum
 
 // 注册函数
 StreamTableEnvironment tEnv = ...
-tEnv.registerFunction("wAvg", new WeightedAvg());
+tEnv.createTemporarySystemFunction("wAvg", new WeightedAvg());
```

Comment:
```suggestion
tEnv.createTemporarySystemFunction("wAvg", WeightedAvg.class);
```

Reference: https://github.com/apache/flink/blob/9d92104f7554a46eeaabf8622ae6670555eb2ad4/docs/content/docs/dev/table/functions/udfs.md?plain=1#L1090

Reply:
in fact both options are possible since there are both methods in `TableEnvironment` interface
https://github.com/apache/flink/blob/9d92104f7554a46eeaabf8622ae6670555eb2ad4/flink-table/flink-table-api-java/src/main/java/org/apache/flink/table/api/TableEnvironment.java#L469-L470
and
https://github.com/apache/flink/blob/9d92104f7554a46eeaabf8622ae6670555eb2ad4/flink-table/flink-table-api-java/src/main/java/org/apache/flink/table/api/TableEnvironment.java#L493

however agree it better to follow the same pattern in both doc versions

Reply:
done